services/management/backup/locations_service_test.go

```diff
@@ -169,3 +169,54 @@ func TestListBackupLocations(t *testing.T) {
 	})
 
 }
+
+func TestRemoveBackupLocation(t *testing.T) {
+	ctx := context.Background()
+	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+	db := reform.NewDB(sqlDB, postgresql.Dialect, reform.NewPrintfLogger(t.Logf))
+
+	svc := NewLocationsService(db)
+
+	req := &backupv1beta1.AddLocationRequest{
+		Name: gofakeit.Name(),
+		PmmClientConfig: &backupv1beta1.PMMClientLocationConfig{
+			Path: "/tmp",
+		},
+	}
+	res1, err := svc.AddLocation(ctx, req)
+	require.NoError(t, err)
+	req.Name = gofakeit.Name()
+	_, err = svc.AddLocation(ctx, req)
```

Comment:
this location is not used anywhere. Any reason to create it? Maybe we should check that this location is not removed, what do you think?